codecs/openjphjs.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
@swederik - is this supposed to go in the codecs module instead?  
@jmhmd - static-dicomweb pulls this in from the codecs module, and uses both the decompressor and the compressor side, although for getting DICOMweb files in the HTJ2K format, it needs the compressor side only for now.

Reply:
The codec I am using is the WASM build from Chris's openjphjs repo. I am not sure if there is a way to separate the compressor/decompressor, but that honestly would be a neat idea as it would presumably make the wasm binary smaller.